Makefile.in

```diff
@@ -155,6 +155,7 @@ C_SRCS := $(srcroot)src/jemalloc.c \
 	$(srcroot)src/thread_event.c \
 	$(srcroot)src/ticker.c \
 	$(srcroot)src/tsd.c \
+	$(srcroot)src/util.c \
```

Comment:
@guangli-dai forgot that we also need to update the MSVC project sources, e.g. in 
https://github.com/jemalloc/jemalloc/blob/dev/msvc/projects/vc2022/jemalloc/jemalloc.vcxproj#L38

See an example in 
https://github.com/jemalloc/jemalloc/commit/8edfc5b1700eab47d64d7cfa6a246ad88f832845

Reply:
opps, will do a patch right now